pkg/kubernetes/client_test.go

```diff
@@ -384,9 +382,10 @@ func TestCountActiveLoadTests(t *testing.T) {
 
 	logger := zap.NewNop()
 	c := NewClient(loadtestClientset.KangalV1().LoadTests(), kubeClientSet, logger)
-	counter, err := c.CountActiveLoadTests(ctx)
+	counter, _, err := c.CountExistingLoadtests()
```

Comment:
I think we are missing some test cases for this function :D 
- Tests for the new counter for load test types
- Tests for all phases
- Negative test when there is error
- Test when the loadtype or phase is unknown